x-pack/plugins/security/public/management/users/edit_user/edit_user_page.tsx

```diff
@@ -378,6 +376,20 @@ export class EditUserPage extends Component<Props, State> {
       return null;
     }
 
+    const hasAnyDeprecatedRolesAssigned = selectedRoles.some(selected => {
```

Comment:
Can we add a test for this? We added one for the role mappings editing, but not here 😬